### ISSUE TITLE
Main promo banner

### ISSUE THREAD

**Issue by schalkneethling** [mdn/yari#1178]
Move the styling of the main promo banner to Yari

- Cleanup SASS to avoid unnecessary duplication
- Implement base banner style
- Implement basic theme
- Document in Storybook

**Comment by peterbe** [comment]
[Ditto :)](https://github.com/mdn/yari/issues/1177#issuecomment-681034470)

**Comment by schalkneethling** [comment]
> [Ditto :)](https://github.com/mdn/yari/issues/1177#issuecomment-681034470)

True, but then again, this is essentially in Yari already and could be used for many things other than subscriptions. So I think it is fine to spend some time one it

**Comment by peterbe** [comment]
Then, what's the promo banner? Perhaps I'm just no understanding the term.

**Comment by schalkneethling** [comment]
> Then, what's the promo banner? Perhaps I'm just no understanding the term.

:) Yeah, we need consistent naming around these things. I was referring to the "sticky" banner that shows up at the bottom of pages aka active-banner.tsx

**Comment by peterbe** [comment]
Ah! The banners definitely need the Fiori-love because it's highly likely to become useful with or without subscriptions in 2020.

**Comment by schalkneethling** [comment]
Resolved by https://github.com/mdn/yari/pull/1225
